tools/topology/sof-tgl-sdw-max98373-rt5682.m4

```diff
@@ -57,6 +59,13 @@ ifelse(PLATFORM, `adl',
 `	define(`SMART_ALH_INDEX', 0x202)
 	define(`SMART_ALH_PLAYBACK_NAME', `SDW2-Playback')
 	define(`SMART_ALH_CAPTURE_NAME', `SDW2-Capture')
+	# Add BT audio offload support
+	define(`BT_PIPELINE_PB_ID', `14') dnl DMIC_PIPELINE_KWD_ID + 1
+	define(`BT_PIPELINE_CP_ID', `15') dnl DMIC_PIPELINE_KWD_ID + 2
```

Comment:
again I think we are confusing pipelines and PCMs ID

Reply:
@plbossart This would seem to follow the convention used e.g. for DMIC. The first free ID is used for pipeline ids and name has "FOO_PIPELINE_BAR_ID". The PCM IDs are picked from the recommendation for soundwire devices and the macros are "FOO_PCM_BAR_ID". So I think this is good.